app/templates/password-recovery.hbs

```diff
@@ -61,9 +61,12 @@
             <ProgressButton type="submit" @loading={{ this.pilasBloquesApi.loading.credentials }}>
               {{t "templates.password-recovery.finish"}}
             </ProgressButton>
-            <PaperButton onClick={{ action this.step.previousStep }}>
-              {{t "templates.password-recovery.back"}}
-            </PaperButton>
+            
+            {{#if this.step.previousStep}}
+              <PaperButton onClick={{ action this.step.previousStep }}>
+                {{t "templates.password-recovery.back"}}
+              </PaperButton>
+            {{/if}}
 
```

Comment:
`this.step.previousStep` estaba siendo undefined y explotaba. No se porque empezo a saltar ahora, pero en produccion parece que tambien es undefined, porque ese boton pareciera que no hace nada.

Solo vi que el boton aparece cuando se vá a la url que llega por mail, el que tiene el token como param:
![image](https://user-images.githubusercontent.com/37090248/207934731-f84b9878-9843-4336-b15f-c2098366906a.png)

Creo que tiene sentido que aca no haya `previousStep`, porque no hay previous step al estar en esta pantalla.

No se si este boton aparece en otras situaciones, pero si es solo aca, tal vez tendria mas sentido borrarlo.